### ISSUE TITLE
L048 applies to IN list

### ISSUE THREAD

**Issue by rnorberg** [sqlfluff/sqlfluff#943]
Rule [`L048`](https://github.com/sqlfluff/sqlfluff/blob/master/src/sqlfluff/core/rules/std/L048.py) results in a linting failure on the following query, which is (in my opinion) formatted correctly:

```sql
SELECT some_column
FROM some_table
WHERE some_other_column IN ('some literal', 'some other literal');
```

Here's a quick way to reproduce the error in bash<sup>[1](#footnote1)</sup>.

```bash
q=$'SELECT some_column\nFROM some_table\nWHERE some_other_column IN (\'some literal\', \'some other literal\');'
echo "$q" | sqlfluff lint - --rules L048
```

```
== [stdin] FAIL
L:   3 | P:  29 | L048 | Missing whitespace after e_literal'
```

## Expected behavior

I expect running `sqlfluff lint` on the query above to return with exit code 0 (no linting failure).

## Observed behavior

Running `sqlfluff lint` on the query above results in a failure state, failing rule L048.

## Version

```bash
$ sqlfluff --version
sqlfluff, version 0.5.1

$ python3 --version
Python 3.6.8
```

## Configuration

Default (no config files).

## Extra info

Here's a slightly more complex query that would make a good test case for rule L048:

```sql
SELECT
    col1,
    'string literal' AS new_column_literal,
    CASE WHEN col2 IN ('a', 'b') THEN 'Y' ELSE 'N' END AS new_column_case
FROM some_table
WHERE col2 IN ('a', 'b', 'c', 'd');
```
<a name="footnote1">1</a>: Please excuse the weird backslash escape sequences - [StackOverflow told me this was how to get a newline in a string](https://stackoverflow.com/a/3182519).

**Comment by rnorberg** [comment]
@alanmcruickshank sorry to crap on your sweet, brand new feature :(

I see you [added L048](https://github.com/sqlfluff/sqlfluff/commit/1e83332f2741d14bea8d142a4abeeb539f6b983b#diff-f5858c90d6ed606375d0d6e6dff094014cf326890133bf4facf85fc1970e6f32) just a couple days ago and @barrywhart reviewed [the PR](https://github.com/sqlfluff/sqlfluff/pull/929). 

I wanted to let you know about this while it's still fresh in your minds. I just recently discovered sqlfluff and I'm a huge fan already. Thank you!

**Comment by alanmcruickshank** [comment]
Great catch, you're totally right, I'll get a fix out in the next fees days. Welcome to the project!

**Comment by alanmcruickshank** [comment]
@rnorberg fix PR is ready, will be released in the next 24 hrs.

**Comment by rnorberg** [comment]
Y'all are awesome!!! Thank you so much!
